src/askContainer/webViews/createSkillWebview.ts

```diff
@@ -118,7 +118,12 @@ export class CreateSkillWebview extends AbstractWebView {
         Logger.verbose(`Calling method: ${this.viewId}.createSmapiResource, args: `, skillId, skillName);
         let profile = Utils.getCachedProfile(this.extensionContext);
         profile = profile ?? DEFAULT_PROFILE;
-        const vendorId = Utils.resolveVendorId(profile);
+        let vendorId: string;
+        try {
+            vendorId = Utils.resolveVendorId(profile);
+        } catch (err) {
+            throw loggableAskError(`Failed to retrieve vendorID for profile ${profile}`, err, true);
```

Comment:
nit: Instead of doing this, we can handle the same behavior in the utility function right? Also, I think this is an edge case and is not in general skill dev flow.